app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,128 @@
+    <!-- Indicators -->
+  <ol class="carousel-indicators">
+    <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+    <li data-target="#myCarousel" data-slide-to="1"></li>
+    <li data-target="#myCarousel" data-slide-to="2"></li>
+    <li data-target="#myCarousel" data-slide-to="3"></li>
+    <li data-target="#myCarousel" data-slide-to="4"></li>
+  </ol>
+
+  <div class="carousel-inner" role="listbox">
+    <div class="item active">
+      <img src="http://www.qualityfootwear.ca/wp-content/uploads/2014/03/park_lane-0079.jpg" alt="Sandals">
+      <div class="carousel-caption">
+        <h3>Sandals</h3>
+        <p>Make your feet happy</p>
+      </div>
+    </div>
+
+    <div class="item">
+        <img src="https://onmogul.s3.amazonaws.com/uploads/product/thumbnail/16569/3cb4f4e1ec.jpg" alt="Scarf">
+        <div class="carousel-caption">
+          <h3>Scarves</h3>
+          <p>Add a pop of color to your look</p>
+        </div>
+    </div>
+
+<!--     <div class="item">
+        <img src="https://jenniferfisherjewelry.com/media/wysiwyg/JFBanner1280x350_3.jpg" alt="Jewelry">
+        <div class="carousel-caption">
+          <h3>Jewelry</h3>
+          <p>Just your style.</p>
+        </div>
+    </div> -->
+
+    <div class="item">
+        <img src="http://www.this-that.co/images/files/category-page/fashion-1.jpg" alt="Bag">
+        <div class="carousel-caption">
+          <h3>Bags</h3>
+          <p>Carry your world, in style</p>
+        </div>
+    </div>
+
+<!--     <div class="item">
+        <img src="http://www.emiliacosi.com/uploads/images/a6908135ab63a95962d556767f7bc410c35637fe.jpg" alt="Bag">
+        <div class="carousel-caption">
+          <h3>Flowers</h3>
+          <p>Beatiful flowers in Kolymbari, Crete.</p>
+        </div>
+    </div> -->
+
+    <div class="item">
+        <img src="https://onmogul.s3.amazonaws.com/uploads/article/thumbnail/20451/fa7047cb4f.jpg" alt="Shoes">
+        <div class="carousel-caption">
+          <h3>Shoes</h3>
+          <p>Your closet will thank you</p>
+        </div>
+    </div>
+
+    <div class="item">
+        <img src="https://jenniferfisherjewelry.com/media/wysiwyg/JFBanner1280x350_5.jpg" alt="Bag">
+        <div class="carousel-caption">
+          <h3>Jewelry</h3>
+          <p>A little bling...or a lot?</p>
+        </div>
+    </div>
+
+  </div>
+
+    <!-- Left and right controls -->
+  <a class="left carousel-control" href="#myCarousel" role="button" data-slide="prev">
+    <span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+    <span class="sr-only">Previous</span>
+  </a>
+  <a class="right carousel-control" href="#myCarousel" role="button" data-slide="next">
+    <span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+    <span class="sr-only">Next</span>
+  </a>
+</div>
+<!-- ######################## -->
+<br><br><br><br>
+
+<div class="row">
+  <h3 class="bycategory">Shop by Category</h3>
```

Comment:
Since you have categories in your database, how could you go about updating this to include all the categories?